.automation/add-to-project-when-labeled

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import sys
+from github import Github
+
+# A mapping between labels and projects to add when they're applied.
+LABEL_PROJECTS = {
+    "infrastructure": "Infrastructure",
+    "amd sev": "SEV",
+    "intel sgx": "SGX",
+    "wasm": "WebAssembly",
+}
+
+# Get inputs from shell
+(token, repository, path) = sys.argv[1:4]
+
+# Initialize repo
+repo = Github(token).get_repo(repository)
+
+# Open Github event JSON
+with open(path) as f:
+    event = json.load(f)
+
+# Determine whether the labels were added or removed
+action = event["action"]
```

Comment:
Is event in scope? I see its first use inside the block starting on line 23.

Reply:
Yes. Python scope is weird...